features/raadpleeg-verblijfplaats-met-periode/gba/overzicht-gba.feature

```diff
@@ -0,0 +1,330 @@
+  In tegenstelling tot verblijfplaatsen met bekend datum aanvang adreshouding/adres buitenland, hoeft de sortering niet overeen te komen met het aflopend sorteren op datum aanvang adreshouding/adres buitenland.
+  Dit is bijvoorbeeld het geval bij een verblijfplaats met datum aanvang adreshouding die in de onzekerheidsperiode ligt van een volgende verblijfplaats met deels onbekende datum aanvang adreshouding en de datum aanvang adreshouding ligt in de gevraagde periode.
+
+  Abstract Scenario: datumTot van de gevraagde periode is gelijk aan de eerste dag van de deels onbekende datum aanvang adreshouding (geen vorige en geen volgende verblijfplaats)
+    Gegeven de persoon met burgerservicenummer '000000024' is ingeschreven op adres 'A1' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | <datum aanvang adreshouding>       |
+    Als gba verblijfplaatsen wordt gezocht met de volgende parameters
+    | naam                | waarde              |
+    | type                | RaadpleegMetPeriode |
+    | burgerservicenummer | 000000024           |
+    | datumVan            | 2007-01-01          |
+    | datumTot            | <datum tot>         |
+    Dan heeft de response verblijfplaatsen met de volgende gegevens
+    | adresseerbaarObjectIdentificatie | straat | datumAanvangAdreshouding     |
+    | 0800010000000001                 | Laan   | <datum aanvang adreshouding> |
+
+    Voorbeelden:
+    | datum aanvang adreshouding | datum tot  |
+    | 20100800                   | 2010-08-01 |
+    | 20100000                   | 2010-01-01 |
+
+  Abstract Scenario: <scenario> (wel vorige en geen volgende verblijfplaats)
+    Gegeven de persoon met burgerservicenummer '000000024' is ingeschreven op adres 'A1' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20100810                           |
+    En de persoon is vervolgens ingeschreven op adres 'A2' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20160000                           |
+    Als gba verblijfplaatsen wordt gezocht met de volgende parameters
+    | naam                | waarde              |
+    | type                | RaadpleegMetPeriode |
+    | burgerservicenummer | 000000024           |
+    | datumVan            | <datum van>         |
+    | datumTot            | <datum tot>         |
+    Dan heeft de response verblijfplaatsen met de volgende gegevens
+    | adresseerbaarObjectIdentificatie | straat | datumAanvangAdreshouding     |
+    | 0800010000000001                 | Laan   | <datum aanvang adreshouding> |
+
+    Voorbeelden:
```

Comment:
in de eerste 2 voorbeelden ligt de periode gedeeltelijk in de onzekerheidsperiode van verblijf op A2. Dan moet je naast verblijfplaats A1 ook verblijfplaats A2 leveren.
In het 3e voorbeeld ligt de periode na de onzekerheidsperiode van verblijf op adres A2. Dan moet je wel verblijfplaats A2 leveren, maar niet verblijfplaats A1

Reply:
> in de eerste 2 voorbeelden ligt de periode gedeeltelijk in de onzekerheidsperiode van verblijf op A2. Dan moet je naast verblijfplaats A1 ook verblijfplaats A2 leveren.

Als dit is wat we willen, dan klopt de rule van regel 175 niet. Daar wordt gezegd dat bij deels onbekend datum aanvang adreshouding/adres buitenland de eerste dag van de onzekerheidsperiode als datum aanvang adreshouding/adres buitenland wordt gehanteerd. Als dit wordt gezegd, betekent dit dan niet dat datum aanvang volgende adreshouding/adres buitenland van de vorige verblijfplaats dan gelijk moet zijn aan de eerste dag van de onzekerheidsperiode?

In jouw geval is datum aanvang volgende adreshouding/adres buitenland van de vorige verblijfplaats gelijk aan de dag na de onzekerheidsperiode.

Reply:
Volgens mij is het geen kwestie van wat we (of ik) willen, maar van juiste interpretatie van de registratie. 

De rule op regel 175 beschrijft hoe wordt omgegaan met onbekende datum aanvang. Die zegt niks over hoe om te gaan met onbekende aanvang volgende verblijf.
De rule is denk ik wel correct, alleen moet er een andere rule bij komen die gaat over onbekende aanvang volgende

Reply:
> Volgens mij is het geen kwestie van wat we (of ik) willen, maar van juiste interpretatie van de registratie.
>
Daarom vraag ik mij af of (alle) bewoning rules van toepassing zijn voor verblijfplaats historie. Weliswaar zelfde bron, maar verschillende view.

Als in de eerste twee voorbeelden ook het verblijf op A2 moet worden geleverd, dan zeggen we hiermee dat een persoon in een periode 2 verblijfplaatsen kan hebben. Volgens mij hadden wij naar aanleiding van de rule met betrekking tot geheel onbekend aanvang adreshouding gezegd dat een persoon geen twee verblijfplaatsen kan hebben.
Dit is (denk ik) het meest duidelijk als de gevraagde periode 1 dag lang is (equivalent aan raadpleeg op peildatum) en de gevraagde periode valt in de onzekerheidsperiode van de volgende adreshouding, dan wordt voor de persoon op peildatum twee verblijfplaatsen geleverd.